targets/PythonSdk/source/uploadPython.ps1

```diff
@@ -0,0 +1,26 @@
+Param(
```

Comment:
I don't see any reason this needs to be powershell.
I'd prefer bash.
I'm trying to consolidate all the Jenkins scripts to bash.

Reply:
doesn't NEED to be powershell, but it's 1.) what I know 2.) works

just tried converting this to a .sh and ran into various problems (powershell has to launch git bash which just closes and doesn't tell you what's wrong, launching from git bash doesn't allow me to type in my password for pypi? at least that's what was happening yesterday, launching from ubuntu is saying it doesn't understand my cd commands). Uploaded an equivalent .sh file for you to look at.

Reply:
Adding a comment to continue this work for Jekins. There are several issues with Git Bash (pypi doesn't accept username) and Ubuntu on Windows (pip isn't installed?) that are acting SUPER lame right now